packages/lib-classifier/src/components/Classifier/components/SlideTutorial/SlideTutorial.spec.js

```diff
@@ -44,4 +46,25 @@ describe('SlideTutorial', function () {
     const wrapper = shallow(<SlideTutorial stepWithMedium={{ step, medium }} />)
     expect(wrapper.find(Media)).to.have.lengthOf(1)
   })
+
+  it('should only render the Heading on the first step', function () {
+    const wrapper = shallow(<SlideTutorial activeStep={0} stepWithMedium={{ step, medium }} isFirstStep />)
+    expect(wrapper.find(Heading)).to.have.lengthOf(1)
+    wrapper.setProps({ activeStep: 1, isFirstStep: false })
```

Comment:
Minor point about the component API, but it seems weird to me to that it's possible to pass in a combination like `activeStep: 1, isFirstStep: true`.

Reply:
Yeah, I see how that is a bit confusing. The activeStep prop is only used in constructing the aria-label for the radio buttons to be read aloud "Go to step 3" for instance.

Reply:
Also the alt text for the media, "Tutorial step 3", which I'm not sure how useful that is.